src/github/branch.ts

```diff
@@ -77,14 +79,23 @@ export const deleteBranchWebhookHandler = async (context: WebhookContext, jiraCl
 		return;
 	}
 
-	context.log.info(`Deleting branch for repo ${context.payload.repository?.id} with ref ${context.payload.ref}`);
+	const gitHubAppId = context.gitHubAppConfig?.gitHubAppId;
+	const gitHubProduct = getCloudOrServerFromGitHubAppId(gitHubAppId);
+
+	context.log.info(
```

Comment:
Hi, there're quite a lot of changes in the webhook related handlers, I assume they are all for the main purpose of adding a product field in the log?

If so, we can do it in the GithubWebhookMiddlware at [this line](https://github.com/atlassian/github-for-jira/blob/7b3e3f5de2af73f01a331595acc55724e3a8089a/src/middleware/github-webhook-middleware.ts#L106), like `context.log.addFields({...product})`, so that the change is just in one place. 

Do you think it is okay to do that?

Reply:
Wait, haha, I see you already added that in the middleware, so I guess if just for adding the `product` in the logs fields, we don't have to do it in all these handlers? Or is there a reason that the `product` set in the middleware doesn't work here?